bin/check-redis-list-length.rb

```diff
@@ -63,19 +63,38 @@ class RedisListLengthCheck < Sensu::Plugin::Check::CLI
          description: 'Redis list KEY to check',
          required: true
 
+  option :threshold,
```

Comment:
sounds, good I will keep my eye out for a fix, also it looks like you still have some rubocop changes to make as well.